test/integration.js

```diff
@@ -1,15 +1,17 @@
 "use strict";
-const path = require("path");
-const fs = require("fs");
+
+const chai = require("chai");
+const chaiAsPromised = require("chai-as-promised");
 const childProcess = require("child_process");
+const fs = require("fs");
 const mkdirp = require("mkdirp");
+const path = require("path");
 const rimraf = require("rimraf");
-const chai = require("chai");
-const chaiAsPromised = require("chai-as-promised");
+
```

Comment:
Yeah, please don't include these changes here

Reply:
No problem. I've disabled it in my IDE for this project now. Sorry about that.